src/assets/scripts/base.js

```diff
@@ -27,224 +27,32 @@ function getParameterByName(name) {
   return results === null ? "" : decodeURIComponent(results[1].replace(/\+/g, " "));
 }
 
-/**
- * Perform a search and display the result
- *
- * @param q  String     The user's search query
- */
-function doSearch(q) {
-  // reset search
-  $(".result").empty();
-  $(".searchinstructions").hide();
-
-  var limit = 100;
-  $("#searchInput").val(q);
-  // assemble the big query object for ElasticSearch
-  var postData = {
-    "from": 0,
-    "size": limit,
-    "sort": ["_score"],
-    "_source": {
-      "excludes": ["text", "breadcrumb_*"]
-    },
-    "query": {
-      "function_score": {
-        "query": {
-          "simple_query_string": {
-            "fields": ["title^5", "uri^5", "text"],
-            "default_operator": "AND",
-            "query": q
-          }
-        },
-        "functions": [
-          // changelog ranks low
-          {
-            "filter": {"term": {"breadcrumb_1": "changes"}},
-            "weight": 0.1
-          },
-          // REST API spec ranks low
-          {
-            "filter": {"term": {"breadcrumb_1": "api"}},
-            "weight": 0.3
-          },
-          // Blog posts rank a bit lower
-          {
-            "filter": {"term": {"breadcrumb_1": "blog"}},
-            "weight": 0.8
-          },
-          // Entries rank lower the older they are
-          {
-            "linear": {
-              "date": {
-                "scale": "100d",
-                "decay": 0.2
-              }
-            }
-          }
-        ]
-      }
-    },
-    "highlight" : {
-      "fields" : {
-        "body" : {
-          "type": "unified",
-          "number_of_fragments" : 1,
-          "no_match_size": 200,
-          "fragment_size" : 150
-        },
-        "title": {
-          "type": "unified",
-          "number_of_fragments" : 1
-        }
-      }
-    },
-    "suggest": {
-      "phrase-suggester": {
-        "text": q,
-        "phrase": {
-          "field": "text.trigram",
-          "size": 1,
-          "gram_size": 3,
-          "direct_generator": [
-            {
-              "field": "text.trigram",
-              "suggest_mode": "popular",
-              "min_word_length": 3,
-            }
-          ],
-        }
+/* Initialize Inkeep widget for the Search Box */
+var inkeepWidget = null;
+$(function() {
+  inkeepWidget = Inkeep().embed({
+    componentType: "SearchBar", // required
+    targetElement: document.getElementById("searchInputMobile"), // required
+    properties: {
+      stylesheetUrls: ["/css/inkeep.css"],
+      baseSettings: {
+        apiKey: "97dd6334a2ba9121a71ec3e88216e593670153eba46954b0", // required
```

Comment:
Can I assume this is okay to have public, and the widget is limited to only be used on `docs.giantswarm` (and maybe `localhost`)?

Reply:
yeah that is what I asked last week to Nick, it seems there is no problem with that 
https://gigantic.slack.com/archives/C05UY4GNN7M/p1700491629729709?thread_ts=1700490386.730039&cid=C05UY4GNN7M